gateway/api/ray.py

```diff
@@ -229,6 +233,10 @@ def create_ray_cluster(
     dyn_client = DynamicClient(k8s_client)
     raycluster_client = dyn_client.resources.get(api_version="v1", kind="RayCluster")
     response = raycluster_client.create(body=cluster_data, namespace=namespace)
+    print(cluster_data)
```

Comment:
Right now I'm just taking an overview of the project, @akihikokuroda . Let me know if you want that I enter to see specific things from the code like this one.

Reply:
I'm debugging the CI test error now.  This is added just for it.  I'll take out when it's done.  Thanks!